x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/signals_filter_group/index.tsx

```diff
@@ -11,41 +11,41 @@ export const FILTER_OPEN = 'open';
 export const FILTER_CLOSED = 'closed';
 export type SignalFilterOption = typeof FILTER_OPEN | typeof FILTER_CLOSED;
 
-export const SignalsTableFilterGroup = React.memo(
-  ({
-    onFilterGroupChanged,
-  }: {
-    onFilterGroupChanged: (filterGroup: SignalFilterOption) => void;
-  }) => {
-    const [filterGroup, setFilterGroup] = useState(FILTER_OPEN);
-
-    const onClickOpenFilterCallback = useCallback(() => {
-      setFilterGroup(FILTER_OPEN);
-      onFilterGroupChanged(FILTER_OPEN);
-    }, [setFilterGroup, onFilterGroupChanged]);
-
-    const onClickCloseFilterCallback = useCallback(() => {
-      setFilterGroup(FILTER_CLOSED);
-      onFilterGroupChanged(FILTER_CLOSED);
-    }, [setFilterGroup, onFilterGroupChanged]);
-
-    return (
-      <EuiFilterGroup>
-        <EuiFilterButton
-          hasActiveFilters={filterGroup === FILTER_OPEN}
-          onClick={onClickOpenFilterCallback}
-          withNext
-        >
-          {i18n.OPEN_SIGNALS}
-        </EuiFilterButton>
-
-        <EuiFilterButton
-          hasActiveFilters={filterGroup === FILTER_CLOSED}
-          onClick={onClickCloseFilterCallback}
-        >
-          {i18n.CLOSED_SIGNALS}
-        </EuiFilterButton>
-      </EuiFilterGroup>
-    );
-  }
-);
+interface Props {
+  onFilterGroupChanged: (filterGroup: SignalFilterOption) => void;
+}
+
+const SignalsTableFilterGroupComponent: React.FC<Props> = ({ onFilterGroupChanged }) => {
```

Comment:
As above, why the change to `React.FC` and then wrapping in `React.memo` below?

Also, in this instance, there is no `displayName` set for either `SignalsTableFilterGroupComponent` or `SignalsTableFilterGroup` and I'm seeing no linter error. If I move `SignalsTableFilterGroupComponent` back to `React.memo` I then see the _"Component definition is missing displayName"_ linter error. Any idea why we're not seeing errors here?

Reply:
for components defined with `const` the variable name is propagated automatically as a component name, so we don't have to do it manually